database/factories/CategoryFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Database\Factories;
+use App\Models\Category;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Category>
+ */
+class CategoryFactory extends Factory
+{
+    protected $model = Category::class;
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'title'=>$this->faker->title,
```

Comment:
Faker title generates things like "DR", "MRS", ... 
It doesn't make sense for here
Can you find another faker method that is more suitable for here?

Reply:
fixed sir